packages/framework/src/Markdown/Contracts/FrontMatter/SubSchemas/AuthorSchema.php

```diff
@@ -15,5 +15,7 @@ interface AuthorSchema extends BlogPostSchema
         'name' => 'string',
         'username' => 'string',
         'website' => 'string',
+        'bio' => 'string',
```

Comment:
Also support 'biography'

Reply:
Would require a factory probably

Reply:
Todo added in https://github.com/hydephp/develop/commit/af6fa4317ba8a05cb01bfbe579af75f25e736bfe